jans-cedarling/flask-sidecar/main/config.py

```
@@ -43,23 +44,24 @@ class BaseConfig:
-    )
-    if CEDARLING_BOOTSTRAP_CONFIG_FILE == "None":
-        logger.info(
-            "Cedarling bootstrap file not found, falling back to environment variables"
-        )
-    else:
+
+    @classmethod
+    def load_bootstrap(cls) -> None:
+        path = os.getenv("CEDARLING_BOOTSTRAP_CONFIG_FILE")
+        if not path:
+            raise RuntimeError("CEDARLING_BOOTSTRAP_CONFIG_FILE is not set")
```

Ruff (0.15.11)

[warning] Avoid specifying long messages outside the exception class

(TRY003)